pkg/activitypub/service/monitoring/monitoring_test.go

```diff
@@ -31,6 +31,9 @@ import (
 const (
 	storeName       = "monitoring"
 	tagNotConfirmed = "not_confirmed"
+
+	webfingerPath    = "/.well-known/webfinger"
```

Comment:
This is already defined in https://github.com/trustbloc/orb/blob/06ac36232fbaaeb13597a11d3f968a15d61e4987/pkg/discovery/endpoint/restapi/operations.go#L27